<launch>

	  <node name="people_detection_node" pkg="people_detection" type="people_detection_node" />
    	     	<param name="rgbcamera" type="string" value="kinect"/>
    	    	<param name="cameralink" type="string" value="/camera_rgb_optical_frame"/>
		<param name="frameoutcondition" type="int" value="12"/>
		<param name="frameincondition" type="int" value="3"/>
		<param name="frameentrylifetime" type="int" value="5"/>
		<param name="detect_length" type="double" value="3.5"/>
  <!--	</node> -->


</launch>
